scripts/package.sh

```diff
@@ -1,5 +1,14 @@
-yarn run build-react
-NODE_ENV=production electron-forge package --platform=win32
-pushd out/wallet-recovery-wizard-darwin-x64/wallet-recovery-wizard.app/Contents/Resources/app
-rm yarn.lock && yarn install --ignore-engines
-popd
\ No newline at end of file
+#!/usr/bin/env bash
+
+./node_modules/.bin/electron-builder -m
+
+docker run --rm -ti \
+ --env-file <(env | grep -iE 'DEBUG|NODE_|ELECTRON_|YARN_|NPM_|CI|CIRCLE|TRAVIS_TAG|TRAVIS|TRAVIS_REPO_|TRAVIS_BUILD_|TRAVIS_BRANCH|TRAVIS_PULL_REQUEST_|APPVEYOR_|CSC_|GH_|GITHUB_|BT_|AWS_|STRIP|BUILD_') \
+ --env ELECTRON_CACHE="/root/.cache/electron" \
+ --env ELECTRON_BUILDER_CACHE="/root/.cache/electron-builder" \
+ -v "${PWD}":/project \
+ -v "${PWD##*/}"-node-modules:/project/node_modules \
```

Comment:
why does `node_modules` get a separate volume?

Reply:
Didn't give it much scrutiny. Pulled straight from docs: https://www.electron.build/multi-platform-build#docker

Reply:
Gotcha